contracts/external/cw-payroll/README.md

```diff
@@ -0,0 +1,47 @@
+# CW Payroll
+
+This contract enables the creation of native && cw20 token streams, which allows a cw20 payment to be vested continuously over time. This contract is forked off of [cw20-streams](https://github.com/CosmWasm/cw-tokens/tree/main/contracts/cw20-streams) to enable additional features required by DAOs for payroll. Key items being: Admin, configurations for vesting, allowing external parties to distribute & more.
+
+## Instantiation
+
+To instantiate a new instance of this contract you must specify a contract owner.
```

Comment:
This "owner" is optional right? If so, that's definitely worth noting. Might be nice to give some example JSON for this and maybe an example CLI command?

```sh
junod tx wasm instantiate <code-id> '{"admin": "juno12xyz..."}'  --label "cw-payroll contract" --from <your-key>
```